confobo/controllers/voting.py

```diff
@@ -20,4 +24,4 @@ def vote(user: User, choice: int, event: Event) -> bool:
     if LOWEST_VOTE <= choice <= HIGHEST_VOTE:
         return voting.save_vote(user, choice, event)
     else:
-        return False
+        raise BadVoteValueError('You can give from {} to {} stars, not {}'.format(LOWEST_VOTE, HIGHEST_VOTE, choice))
```

Comment:
Either we want to throw exceptions, or return bool values. I see no reason to mix these approaces between modules.
THB I'm not really fan of exceptions and stuff and I think it has to be discussed on the next meeting.